ee/dtr/admin/configure/deploy-caches/simple-kube.md

```diff
@@ -0,0 +1,229 @@
+    └── dtr.cert.pem    # DTR CA certificate
+```
+
+### Create the DTR Cache certicates
+
+The DTR cache will be deployed with a TLS endpoint. For this you will need to generate
+a SSL ceritificate and key from a certificate authority. Depending on how you would
+ like to expose the DTR Cache it will depend on the SANs required for this certificate.
+
+For example:
+
+  - If you are deploying the DTR Cache with an 
+    [Ingress Object](https://kubernetes.io/docs/concepts/services-networking/ingress/) 
+    you will need to use the FQDN of your `Ingress Object` as part of your certificate. 
+  - If you exposing the DTR cache through a Kubernetes
+    [Cloud Provider](https://kubernetes.io/docs/concepts/services-networking/#loadbalancer)
+    then you will need the external Loadbalancer address as part of your certificate. 
+  - If you are exposing the DTR Cache through a 
+    [Node Port](https://kubernetes.io/docs/concepts/services-networking/#nodeport)` or a
+    `Host Port]` you will need to use a Node's FQDN as a SAN in your certificate. You could use
+    a Kubernetes `NodeScheduler` to pin the node the DTR cache is deployed on.
+
+On your workstation, create a directory called `certs`. Within here place the newly 
+created certificate `cache.cert.pem` and key `cache.key.pem` for your DTR cache. Also
+ place the certificate authority (including any intermedite certificate authorities)
+ of the certificate from your DTR deployment. This could be sourced from curl.
+`curl -s https://<dtr-fqdn>/ca -o certs/dtr.cert.pem`.
+
+### Create the DTR Config
+
+This is the configuration file of the registry component of the DTR Cache. This yaml
+should be updated with the relevant `cache-fqdn` and `dtr-fqdn`.
+
+```
+version: 0.1
+log:
+  level: info
+storage:
+  delete:
+    enabled: true
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 0.0.0.0:443
+  secret: generate-random-secret
+  host: https://<external-fqdn-dtrcache> # Could be Ingress / LB / Host
+  tls:
+    certificate: /certs/cache.cert.pem
+    key: /certs/cache.key.pem
+middleware:
+  registry:
+      - name: downstream
+        options:
+          blobttl: 24h
+          upstreams:
+            - https://<dtr-url>
+          cas:
+            - /certs/dtr.cert.pem
+```
+
+With this configuration, the cache fetches image layers from DTR and keeps
+a local copy for 24 hours. After that, if a user requests that image layer,
+the cache fetches it again from DTR.
+
+The cache is configured to persist data inside its container.
+If something goes wrong with the cache service, Docker automatically redeploys a
+new container, but previously cached data is not persisted.
+You can [customize the storage parameters](/registry/configuration.md#storage),
+if you want to store the image layers using a persistent storage backend.
+
+### Create Kubernetes Resources
+
+Create a Kubernetes namespace to logical seperate all of our DTR cache components.
```

Comment:
For clarity should add directions about either sourcing the client bundle corresponding to your UCP cluster, or should be on the UCP cluster.